rules/windows/defense_evasion_enable_inbound_rdp_with_netsh.toml

```diff
@@ -14,6 +14,43 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Remote Desktop Enabled in Windows Firewall"
+note = """## Triage and analysis
+
+### Investigating Remote Desktop Enabled in Windows Firewall
+
+Microsoft Remote Desktop Protocol (RDP) is a proprietary Microsoft protocol that enables remote connections to other
+computers, typically over TCP port 3389.
+
+Attackers use RDP in operations to conduct their actions interactively. Ransomware operators heavily use RDP to access
+victim servers, often using privileged accounts.
+
+This rule detects a Windows Firewall inbound rule being added, which will allow RDP traffic to the host using the
+`netsh.exe` utility.
+
+#### Possible investigation steps:
+
+- Identify the user account which performed the action and whether it should perform this kind of action.
+- Contact the user to check if it is aware of the operation.
+- Investigate script execution chain (parent process tree).
+- Investigate other alerts related to the user/host in the last 48 hours.
+- Check whether it makes sense to enable RDP to this host and the host role in the environment.
+- Check if the server is directly exposed to the internet.
+- Check whether privileged accounts have accessed the host in a short period after the modification.
+
+### False Positive Analysis
+
+- This mechanism can be used legitimately. Check whether the user should be performing this kind of activity, is aware
```

Comment:
```suggestion
- This `netsh.exe` utility can be used legitimately. Check whether the user should be performing this kind of activity, whether the user is aware
```

Reply:
Probably want to do something similar to [this](https://github.com/elastic/detection-rules/pull/1836/files#r826216812).